usr/share/rear/layout/prepare/GNU/Linux/110_include_lvm_code.sh

```diff
@@ -67,10 +66,35 @@ create_lvmgrp() {
 
     local vg=${vgrp#/dev/}
 
+    # If a volume group name is in one of the following lists, it
+    # means that the particular condition is valid for this volume
+    # grup. If it is not in the list, it means that the condition is
```

Comment:
I fixed the typo in comment 'grup' -> 'group' via
https://github.com/rear/rear/commit/02dad206b6c2322d111563e16d22c88331392a12